docs/_pages/releases.md

```diff
@@ -3,8 +3,30 @@ title: "Releases"
 permalink: /releases
 layout: releases
 toc: true
-last_modified_at: 2019-12-30T15:59:57-04:00
+last_modified_at: 2020-05-28T08:40:00-07:00
 ---
+## [Release 0.5.3](https://github.com/apache/hudi/releases/tag/release-0.5.3) ([docs](/docs/0.5.3-quick-start-guide.html))
+
+### Download Information
+ * Source Release : [Apache Hudi 0.5.3 Source Release](https://downloads.apache.org/hudi/0.5.3/hudi-0.5.3.src.tgz) ([asc](https://downloads.apache.org/hudi/0.5.3/hudi-0.5.3.src.tgz.asc), [sha512](https://downloads.apache.org/hudi/0.5.3/hudi-0.5.3.src.tgz.sha512))
+ * Apache Hudi jars corresponding to this release is available [here](https://repository.apache.org/#nexus-search;quick~hudi)
+
+### Migration Guide for this release
```

Comment:
Maybe we can add this. 

0.5.3 is the first hudi release after graduation. As a result, all hudi jars will no longer have "-inubating" in the version name. In all the places where hudi version is referred, please make sure "-incubating" is no longer present.

For example hudi-spark-bundle pom dependency would look like:
 <dependency>
    <groupId>org.apache.hudi</groupId>
    <artifactId>hudi-spark-bundle_2.12</artifactId>
    <version>0.5.3</version>
</dependency>